routers/routes/routes.go

```diff
@@ -467,6 +467,19 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/content", repo.UpdateIssueContent)
 				m.Post("/watch", repo.IssueWatch)
 				m.Combo("/comments").Post(bindIgnErr(auth.CreateCommentForm{}), repo.NewComment)
+				m.Group("/times", func() {
+					m.Post("/add", repo.AddTimeManual)
+					m.Group("/stopwatch", func() {
+						m.Get("/toggle", repo.IssueStopwatch)
```

Comment:
I don't see the `/times/stopwatch/toggle` endpoint anywhere in https://github.com/go-gitea/go-sdk/pull/65. But either way, yes, it is worth it. We don't check CSRF tokens for GET requests.